crates/adapters/bybit/src/http/models.rs

```diff
@@ -182,6 +183,11 @@ pub struct BybitTickerData {
     pub low_price24h: String,
     pub turnover24h: String,
     pub volume24h: String,
+    pub open_interest: Option<String>,
```

Comment:
These need `#[serde(default)]` (like the one you added for `usd_index_price` above) or serialization will fail at runtime when they're missing.